### PR TITLE
.github: add integration_test.yml

### PR DESCRIPTION
**Pull request by pierre**

### CODE REVIEW

File: .github/workflows/integration_test.yml [hunk 1]
```diff
@@ -0,0 +1,259 @@
...
+            database-password: 'postgres'
+            database-port: '5432'
+            docker-compose-file: 'docker-compose.ci.postgresql.yml'
+            java-version: '11'
+            java-distribution: 'temurin'
+      max-parallel: ${{ inputs.max-parallel }}
+    steps:
+      - name: Set environment variables
+        run: |
+          echo '${{ secrets.extra-env }}' | jq -r 'to_entries|map("::add-mask::\(.value|tostring)")|.[]'
```
Comment (sbrossie) [inline .github/workflows/integration_test.yml:94]: Just for my understanding: Are those a map of all our secrets on GH?

File: .github/workflows/integration_test.yml [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+          curl --connect-timeout 10 --max-time 30 -v \
+               -X POST \
+               -u admin:password \
+               -H 'Content-Type: application/json' \
+               -H 'X-Killbill-CreatedBy: GitHub' \
+               -d '{"apiKey": "bob", "apiSecret": "lazar"}' \
+               "http://${KB_ADDRESS}:${KB_PORT}/1.0/kb/tenants"
+      - name: Verify plugin has started
+        run: |
+          sudo snap install yq
```
Comment (sbrossie) [inline .github/workflows/integration_test.yml:195]: Do we use `yq` ?

File: .github/workflows/integration_test.yml [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+      - name: Run tests
+        env:
+          DB_ADAPTER: ${{ matrix.database-adapter }}
+          DB_USER: ${{ matrix.database-user }}
+          DB_PASSWORD: ${{ matrix.database-password }}
+          DB_PORT: ${{ matrix.database-port }}
+        run: |
+          cd killbill-integration-tests
+          bundle install --jobs=4 --retry=3
+          bundle exec rake ${{ inputs.integration_tests_goal }}
```
Comment (sbrossie) [inline .github/workflows/integration_test.yml:220]: Perhaps, we could have a rake task for each plugin, i.e use the `plugin_name` as the name of the rake task?

**Review by sbrossie — APPROVED** [review]
👏

File: .github/workflows/integration_test.yml (hunk 1 above)
Comment (pierre) [reply .github/workflows/integration_test.yml:94]: They are selectively sent, e.g. https://github.com/killbill/killbill-avatax-plugin/blob/d3c90df4a660e81db98bd39ed0a8ad012c07452a/.github/workflows/ci.yml#L28

File: .github/workflows/integration_test.yml (hunk 3 above)
Comment (pierre) [reply .github/workflows/integration_test.yml:220]: We do :-) See https://github.com/killbill/killbill-stripe-plugin/blob/9d86e94a14ea725a784c725f11d2385cd38bfebe/.github/workflows/ci.yml#L19 for instance.

File: .github/workflows/integration_test.yml (hunk 2 above)
Comment (pierre) [reply .github/workflows/integration_test.yml:195]: Interesting... maybe not anymore, let me remove it.

**Pull request merged**